mscore/inspector/inspectorMeasureNumber.cpp

```diff
@@ -23,14 +22,17 @@ namespace Ms {
 InspectorMeasureNumber::InspectorMeasureNumber(QWidget* parent)
    : InspectorTextBase(parent)
```

Comment:
Still need to inherit `InspectorTextBase`? Won't that create two inspector sections for measure number?

Reply:
Looking at the code from inspector dynamic, rehearsal mark, etc., it should not.
And in practice, it also does not ;)